.github/workflows/test.yml

```diff
@@ -160,22 +158,16 @@ jobs:
       run: go test -v ./...
     - name: Make
       run: make
-# FIXME: Windows CI began to fail on Oct 21, 2024.
-# Something seems to have changed between win22/20241006.1 and win22/20241015.1.
-# https://github.com/lima-vm/lima/issues/2775
-#    - name: Smoke test
-#      # Make sure the path is set properly and then run limactl
-#      run: |
-#        $env:Path = 'C:\Program Files\Git\usr\bin;' + $env:Path
-#        Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $env:Path
-#        .\_output\bin\limactl.exe start template://experimental/wsl2
-#      # TODO: run the full integration tests
-#    - name: Debug
-#      if: always()
-#      run: type C:\Users\runneradmin\.lima\wsl2\ha.stdout.log
-#    - name: Debug
-#      if: always()
-#      run: type C:\Users\runneradmin\.lima\wsl2\ha.stderr.log
+    - name: Integration tests
+      run: |
+        Invoke-WebRequest https://github.com/git-for-windows/git/releases/download/v2.46.2.windows.1/PortableGit-2.46.2-64-bit.7z.exe -OutFile git-setup.exe
+        cmd /c 'start /w /b git-setup.exe -o".\git-distro" -y'
+        $env:Path = "$pwd\_output\bin;" + "$pwd\git-distro\usr\bin;" + $env:Path
+        Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $env:Path
+        $env:MSYS2_ENV_CONV_EXCL='HOME_SRC;HOME_DST'
+        $env:HOME_SRC=$(.\git-distro\usr\bin\cygpath.exe "$env:USERPROFILE")
+        $env:HOME_DST="/mnt$env:HOME_SRC"
+        .\git-distro\bin\bash.exe -c "LIMACTL_CREATE_ARGS= ./hack/test-templates.sh templates/experimental/wsl2.yaml"
```

Comment:
Isn't bash preinstalled?

Reply:
It is, at least in system wide Git install,  but when called it prefixes $PATH with the msys2 predefined paths of this bash instance and here we need openssh from this very specific installation. It is a workaround for the current incompatibility with the upstream.

Reply:
Will be removed. Updated runners have compatible version.